spec/index.html

```diff
@@ -740,28 +740,25 @@ <h3>Literals</h3>
       <li>If the literal is a <a>language-tagged string</a>,
         then the literal value is a pair consisting of its <a>lexical form</a>
         and its <a>language tag</a>, in that order.</li>
-      <li>if the literal is a <a>directional language-tagged string</a>, then the literal value is
+      <li>If the literal is a <a>directional language-tagged string</a>, then the literal value is
         a tuple of its <a>lexical form</a>, its <a>language tag</a>, and its <a>base direction</a>,
         likewise in that order.</li>
-
-      <li>If the literal's <a>datatype IRI</a> is in the set of
-        <a>recognized datatype IRIs</a>, let <var>d</var> be the
-        <a>referent</a> of the datatype IRI.
+      <li>If the literal's <a>datatype IRI</a> is handled by an RDF processor,
```

Comment:
I find the wording "an RDF processor" strange, all the more that it is repeated below...
Implicitly, we are referring to the *same* processor all along this paragraph, so replacing this (and the other occurrence) with "the RDF processor" would seem better.

Also, the term "RDF processor" is currently only used twice in RDF-Concepts (a little more after this PR), and only in relation with datatypes. Anywhere else in the spec, we talk about Applications and Implementations. Wouldn't it be better to stick to one of these?
```suggestion
      <li>If the literal's <a>datatype IRI</a> is handled by the implementation,
```

Reply:
I'd be happy to remove "RDF Processor" from RDF Concepts  - it's informal but looks formal; it is only for datatypes.

"RDF processor" is used in RDF Semantics more (5 times) so it looks like another thing that has come over from there.